cypress/support/commands/form.js

```diff
@@ -22,6 +22,8 @@ Cypress.Commands.add('clearField', (fieldName, modal) => {
 Cypress.Commands.add('getStringFieldValue', (fieldName, modal) => {
   cy.log(`getStringFieldValue - fieldName=${fieldName}; modal=${modal}`);
 
+  cy.waitForSaveIndicator();
+
```

Comment:
why was this wait needed here?

Reply:
I think i had a case once where i wrote something in a field, but when i read it, its value was still the old one. What i though _could_ fix that was to just wait for the save indicator to be missing.